Makefile

```diff
@@ -2,12 +2,20 @@ GIT_BRANCH=`git rev-parse --abbrev-ref HEAD`
 USER=`whoami`
 STAGING_URL="https://docs-mongodborg-staging.corp.mongodb.com"
 PRODUCTION_URL="https://docs.mongodb.org/meta"
-STAGING_BUCKET=docs-mongodb-org-staging
+STAGING_BUCKET=docs-mongodb-org-prd-staging
 PRODUCTION_BUCKET=docs-mongodb-org-prod
 
 # "PROJECT" currently exists to support having multiple projects
 # within one bucket. For the manual it is empty.
 PROJECT=meta
+PREFIX=ruby-driver
+STGPREFIX=ruby-driver
```

Comment:
Can we remove lines 11 and 12? They don't appear to be used...

Reply:
removed